lib/uffizzi/services/env_variables_service.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'psych'
+require 'pathname'
+require 'base64'
+
+class EnvVariablesService
+  class << self
+    def substitute_env_variables(compose_file_data)
+      compose_file_data.gsub(/\$\{?([?:\-_A-Za-z0-9]+)\}?/) do |variable|
+        variable_content = variable.match(/[?:\-_A-Za-z0-9]+/).to_s
+        fetch_variable_value(variable_content)
+      end
+    end
+
+    private
+
+    def fetch_variable_value(variable_content)
+      variable_name = variable_content.match(/^[_A-Za-z0-9]+/).to_s
+      variable_value = ENV[variable_name]
+      return variable_value unless variable_value.nil?
```

Comment:
if variable_value.present?

Reply:
`present?` is rails method
`return variable_value if variable_value` would work, but I don't like this style. In my opinion `unless variable_value.nil?` more readable